packages/plugin-expo-app/app.js

```diff
@@ -16,23 +16,29 @@ module.exports = {
       lastState = newState
     })
 
-    let nativeBundleVersion, nativeVersionCode
+    let nativeBundleVersion, nativeVersionCode, bundleVersion, versionCode
 
-    if (Constants.appOwnership === 'standalone') {
+    if (Constants.appOwnership !== 'expo') {
       if (Constants.platform.ios) {
         nativeBundleVersion = Application.nativeBuildVersion
+        bundleVersion = Application.nativeBuildVersion
       }
 
       if (Constants.platform.android) {
         nativeVersionCode = Application.nativeBuildVersion
+        versionCode = Application.nativeBuildVersion
       }
     }
 
     client.addOnSession(session => {
-      if (Constants.manifest?.revisionId) {
-        session.app.codeBundleId = Constants.manifest.revisionId
-      } else if (Constants.manifest2?.extra?.expoClient?.revisionId) {
-        session.app.codeBundleId = Constants.manifest2.extra.expoClient.revisionId
```

Comment:
Is `revisionId` not present at all anymore or just if using EAS Update?

It feels like there's still some value in setting this automatically for people not using EAS Update if that's when it goes missing. Maybe now it could be the default value for the new `codeBundleId` config option?

Reply:
There are next to no docs on `revisionId`, but I've tested this manually and it's not present at all any more, whether running an EAS Build or an EAS Update. It's been replaced by `Updates.updateId` (platform-specific identifier for the update) and `Updates.metadata.updateGroup` (platform-agnostic identifier for the update), and these are only present if the app is running a bundle from an EAS Update (as opposed to an embedded bundle from an EAS Build).

As Expo dropped support for classic builds/updates in SDK 47 I'm not sure it makes sense to leave that code in there or default to `revisionId`.

We also decided to stop short of setting `codeBundleId` automatically based on these new properties because we currently don't support automatic source map uploads for EAS Update, but there is an accompanying docs update with a code example (I've added you as a reviewer on that)

Reply:
> There are next to no docs on revisionId, but I've tested this manually and it's not present at all any more, whether running an EAS Build or an EAS Update

What if you're not using EAS at all, or is that no longer an option?

Reply:
Discussed on slack: you _have_ to be using EAS to use the Expo notifier, otherwise you'd be using the RN notifier instead:

> If you’re using the [bare workflow](https://docs.expo.dev/introduction/managed-vs-bare/#bare-workflow) or you’ve [ejected](https://docs.expo.dev/expokit/eject/) your Expo app, you should follow our [React Native](http://docs.bugsnag.com/platforms/react-native/react-native/) guide.
> http://docs.bugsnag.com/platforms/react-native/expo/

Therefore there's no case where you'd have a `revisionId` in Expo 47+ and be using the Expo notifier